cli/flox/src/commands/list.rs

```diff
@@ -118,55 +126,126 @@ impl List {
     /// e.g. `pip: python3Packages.pip (20.3.4)`
     ///
     /// This is the default mode
-    fn print_extended(mut out: impl Write, packages: &[InstalledPackage]) -> Result<()> {
+    fn print_extended(mut out: impl Write, packages: &[PackageToList]) -> Result<()> {
         for p in packages {
-            writeln!(
-                &mut out,
-                "{id}: {path} ({version})",
-                id = p.install_id,
-                path = p.rel_path,
-                version = p.info.version.as_deref().unwrap_or("N/A")
-            )?;
+            match p {
+                PackageToList::CatalogOrPkgdb(p) => {
+                    writeln!(
+                        &mut out,
+                        "{id}: {path} ({version})",
+                        id = p.install_id,
+                        path = p.rel_path,
+                        version = p.info.version.as_deref().unwrap_or("N/A")
+                    )?;
+                },
+                PackageToList::Flake(p) => {
+                    writeln!(
+                        &mut out,
+                        "{id}: {locked_url}#{flake_attr_path}",
```

Comment:
This might be too long, I was just trying to print something unique

Reply:
I think using the locked URL is too long for the default output. I think you could do this:
- If it's a github ref, summarize to `github:<owner>/<repo>` if the attrpath ends in `default`
- If the attrpath doesn't end in `default`, do the above but add the attrpath
- In all other cases maybe you can just literally say "<flake>" and leave the big output for verbose output modes?

Reply:
I think if that's all we want I'd rather just print out whatever `install_id.flake` is in the manifest

Reply:
Updated to print `install_id.flake`